projects/distributed-tracing/src/shared/dashboard/widgets/trace-detail/trace-detail-widget-renderer.component.scss

```diff
@@ -3,7 +3,7 @@
     flex: 1 1 auto;
     display: flex;
     flex-direction: row;
-    padding: 0 16px;
+    padding: 0 16px 0 0;
```

Comment:
nit: more readable just to specify `padding-right`, IMO

Reply:
I thought about that but I didn't have clear if the padding was stepping on something

Reply:
I'd hope not, but that's fine to leave it, just a tiny nit.

Reply:
I replaced it. Thanks!